websocket_push_session.go

```diff
@@ -229,8 +256,32 @@ func (s *wsPushSession) listen() {
 
 			pushConfig := elemental.NewPushConfig()
 			if err := elemental.Decode(s.encodingRead, data, pushConfig); err != nil {
-				s.close(websocket.CloseUnsupportedData)
-				return
+				if s.handlesErrorEvents() {
```

Comment:
can you exit early? 

```
if !s.handlesErrorEvents() {
    s.close(websocket.CloseUnsupportedData)
    return
}
```

Reply:
Nice catch https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow ✨

Reply:
Done ✅